### PR TITLE
Adding the new icon to README.MD

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Screenshot 2022-04-01 at 15 55 17

Screenshot 2022-04-01 at 15 54 54

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Unfortunately, the icon doesn’t look great if using dark mode 😅 
Screenshot 2022-04-01 at 13 43 05

**Review by johansteyn — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## MongoDB Atlas CLI
 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png" alt="CLI Logo">
```
Comment (gssbzn) [inline README.md:4]: can you try this 
```suggestion
<picture>
    <source srcset="atlas_cli_inverse.png" media="(prefers-color-scheme: dark)">
    <img src="atlas_cli.png">
</picture>
```
not sure if git hub will support such a fancy thing

Comment (gssbzn) [reply README.md:4]: (my url are wrong but I hope you get the idea, img with the normal image, prefers dark for the alternative inverse version of the icon

Comment (andreaangiolillo) [reply README.md]: I have just updated the pr with your proposal but it doesn’t seem to work.

Comment (andreaangiolillo) [reply README.md]: it should be possible thought  https://stackoverflow.com/questions/65413712/changing-readme-md-image-display-conditional-to-github-light-mode-dark-mode

File: README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ## MongoDB Atlas CLI
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png" alt="CLI Logo">
+    <picture align="center">
+        <source srcset="https://user-images.githubusercontent.com/5663078/161268959-797863f3-f8c7-479b-b40d-4eab662de658.png" media="(prefers-color-scheme: dark)">
+        <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png">
+    </picture>
```
Comment (gssbzn) [inline README.md:7]: doesn't seem to work but found some GH specific alternative https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

**Comment by dianchenghu** [comment]
The dark mode is fine. We actually have two icons, one for Atlas CLI and one for MongoDB CLI. The current size is too big. I just exported two icons with a smaller size.  @boooczek should we have two icons or one?
atlas_cli
mdb_cli

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**